packages/ui/core-components/src/lib/unsorted/viz/table/Column.svelte

```diff
@@ -81,7 +80,7 @@
 		blue: ['white', 'hsla(198, 56%, 56%,1)']
 	};
 
-	let colorPalette;
+	export let colorPalette;
```

Comment:
Why does this need to be a prop? If it does for whatever reason, we need docs for it.

Reply:
this is used only internally - I added `export` to avoid some browser console warnings, but it's not needed for a user. Will remove